packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -300,8 +353,26 @@ export const createBundle = async ({
                         existsSync(path.join(extendsTemplate, item))
                     )
                 }
+
+                // pwa-kit package versions are not listed as dependencies in package.json
+                // when a bundle is using template extensibility
+                const nestedPwaKitDependencies: {[key: string]: string} = {}
+                if (ccExtensibility.extends) {
```

Comment:
```suggestion
                // pwa-kit package versions are not listed as dependencies in package.json
                // when a bundle is using template extensibility
                const nestedPwaKitDependencies: {[key: string]: string} = {}
```
Template extensibility is the most likely scenario where we'd see PWA Kit packages as nested dependencies, but it's not a hard requirement. We should always check the nested deps!